### DISCUSSION TITLE
Animations in Team Page of amplication site

### DISCUSSION THREAD

**Discussion by thegeekyb0y** [amplication/amplication#2222] [Ideas] [answered]
I have some ideas on howering those profile cards on webpage amplication/team . Let me know if I can show some demo of animations I can do in that page.

**Comment by yuval-hazaz** [comment, marked as answer]
Thanks @thegeekyb0y 
Sounds like a very cool idea, maybe a little spooky 👻
we would love to see an example

**Reply by thegeekyb0y** [reply]
The thing you did with the section "What we do" in amplication.com , the transition of ease in out in 0.3 seconds, this is the same thing you should do with profile cards in that team webpage to look frontend more good. 


.feature__card {
   transition: 0.3s ease-in-out;
    transition-property: all;
    transition-duration: 0.3s;
    transition-timing-function: ease-in-out;
    transition-delay: 0s;
}

**Reply by yuval-hazaz** [reply]
thanks, we will try that
